### DISCUSSION TITLE
Add a profile for zed

### DISCUSSION THREAD

**Discussion by dmitryvakulenko** [netblue30/firejail#6541] [Q&A] [answered]
### Description

I try to make profile for [Zed](https://zed.dev/). Everything fine except one thing - when I try to open project through Ctrl-O I get error "ZBus Error: I/O error: failed to read from socket". But system "Open Folder" dialog should be shown. Looks like application has not access to some dbus socket but I can't find out which one. My profile attached. I tried to remove almost all limitation but there was no any effect. Other application works fine using same profiles.

### Steps to Reproduce

Launch Zed using attached profile and press Ctrl-O.

### Expected behavior

System dialog window "Open folder" will appear.

### Actual behavior
Got error "ZBus Error: I/O error: failed to read from socket" and no dialog window.

### Behavior without a profile
Everything fine - dialog are shown.

### Environment
Linux Mint Cinnamon 22 Wilma.
Kernel 6.8.0-45
firejail version 0.9.72

### Checklist
- [X] The issues is caused by firejail (i.e. running the program by path (e.g. `/usr/bin/vlc`) "fixes" it).
- [ ] I can reproduce the issue without custom modifications (e.g. globals.local).
- [ ] The program has a profile. (If not, request one in `https://github.com/netblue30/firejail/issues/1139`)
- [ ] The profile (and redirect profile if exists) hasn't already been fixed [upstream](https://github.com/netblue30/firejail/tree/master/etc).
- [X] I have performed a short search for similar issues (to avoid opening a duplicate).
  - [ ] I'm aware of `browser-allow-drm yes`/`browser-disable-u2f no` in `firejail.config` to allow DRM/U2F in browsers.
- [X
[zed.profile.gz](https://github.com/user-attachments/files/17686982/zed.profile.gz)
] I used `--profile=PROFILENAME` to set the right profile. (Only relevant for AppImages)

**Comment by powerman** [comment]
Any progress on this?

**Comment by kmk3** [comment, marked as answer]
Duplicate of #1139
